tests/cpp/test_gpu2.cpp

```diff
@@ -2706,21 +2689,28 @@ TEST_F(NVFuserTest, FusionWelfordShmoo_CUDA) {
           //       with half precision. skipping too large volumes for half for
           //       nwo might need further numerical experiments to re-design
           //       this.
-          if (rdim > 32768 &&
+          if (rsize > 32768 &&
               (dtype == DataType::Half || dtype == DataType::BFloat16)) {
             continue;
           }
-          // Shmoo tests can occupy a lot of memory due to allocating many
-          // different tensor sizes. So in order to avoid an OOM during this
-          // test, we manually clear the allocator after it's reached a certain
-          // threshold.
-          maybeClearAllocator();
-          testWelford(dtype, axis, odim, rdim);
+          for (const auto& isize : {160, 320}) {
+            for (const auto& raxis : {0, 1}) {
```

Comment:
FYI, having four nested loops is usually a good sign of needing testing::Combine. The if check on Line 2692 can be turned into a GTEST_SKIP.

Reply:
good point! changed to use `testing::Combine` with `GTEST_SKIP`